common/v2/features/SendAssets/components/validators/validators.ts

```diff
@@ -0,0 +1,33 @@
+import { isValidHex, gasPriceValidator, gasLimitValidator } from 'v2/libs/validators';
+
+export function validateGasPriceField(value: string): string | undefined {
+  let error;
+  if (!gasPriceValidator(value)) {
+    error = 'Gas price must be a valid number.';
+  }
+  return error;
+}
```

Comment:
We can simplify the syntax of these validation functions, for example:
```ts
export function validateGasPriceField(value: string) {
  if (!gasPriceValidator(value)) {
    return 'Gas price must be a valid number.';
  }
}
```

Reply:
done